Dockerfile.dapper

```diff
@@ -4,7 +4,11 @@ FROM ubuntu:16.04
 ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPER_ENV=TAG \
     DAPPER_SOURCE=/go/src/github.com/rancher/submariner DAPPER_DOCKER_SOCKET=true \
-    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} LINT_VERSION=v1.16.0 DAPPER_OUTPUT=output
+    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} DAPPER_OUTPUT=output \
+    LINT_VERSION=v1.17.1 \
+    HELM_VERSION=v2.14.1 \
```

Comment:
You're still bumping the helm version (I believe skitt was asking to separate on a different PR).

Reply:
> You're still bumping the helm version (I believe skitt was asking to separate on a different PR).

I think he requested the environment variables for versions to be on separate lines.

Reply:
> > You're still bumping the helm version (I believe skitt was asking to separate on a different PR).
> 
> I think he requested the environment variables for versions to be on separate lines.

Yes, but I also like changes to be isolated ;-). This PR should only be about kubefed integration, and version bumps unrelated to that should be separate IMO. If the kubefed integration part requires version bumps, that should be documented in the commit message, ideally with a link to the changelog for the new version (see #82 for Helm).

Reply:
Yes you are right i have pushed too many changes. Will revert the versions and open a separate PR.

Reply:
Done!